docs/app/docs/devbox_global.md

```diff
@@ -82,33 +97,22 @@ Make sure to add this hook before any other hooks that use your global packages.
 Add the following command to your `~/.zshrc` file:
 
 ```bash
-eval "$(devbox global shellenv)"
+eval "$(devbox global shellenv --init-hook)"
 ```
 
 ### Fish
 
 Add the following command to your `~/.config/fish/config.fish` file:
 
 ```bash
-devbox global shellenv | source
+devbox global shellenv --init-hook | source
 ```
 
+## Sharing Your Global Config with Git
 
-## Sharing Your Global Config
-
-Your global `devbox.json` will be stored in `$XDG_DATA_HOME/devbox/global/default/devbox.json`. If `$XDG_DATA_HOME` is not set, it will default to `~/.local/share/devbox/global/default/devbox.json`.
-
-If you want to share your configuration with across machines or with other users, you can copy this file to a git repository or host it online.  You can then download and set the config as your global profile using `devbox global pull <path> | <url>`.
-
-```bash
-# Load the global config from a file
-
-devbox global pull /path/to/devbox.json
+You can use Git to synchronize your `devbox global` config across multiple machines using `devbox global push <remote>` and `devbox global pull <remote>`.
 
-# Load the global config from a github repo
-
-devbox global pull https://raw.githubusercontent.com/org/repo/branch/path/to/devbox.json
-```
+Your global `devbox.json` and any other files in the Git remote will be stored in `$XDG_DATA_HOME/devbox/global/default/devbox.json`. If `$XDG_DATA_HOME` is not set, it will default to `~/.local/share/devbox/global/default/devbox.json`.
```

Comment:
"stored in `$XDG_DATA_HOME/devbox/global/default/devbox.json`." 
This should be a directory, not a file, to make the sentence make sense.